webrtc.html

```diff
@@ -1767,18 +1767,17 @@
                 attribute represents the local
                 <code><a>RTCSessionDescription</a></code> that was successfully
                 negotiated the last time the <code>RTCPeerConnection</code>
-                transitioned into the stable state plus any local candidates
-                that have been generated by the <a>ICE Agent</a> since the offer
-                or answer was created.</p>
+                transitioned into the stable state.</p>
+                <p>Note that the value of
+                <code>currentLocalDescription.sdp</code> need not match the
+                value of <code>localDescription.sdp</code>.</p>
```

Comment:
`localDescription` will always be either `currentLocalDescription` or `pendingLocalDescription`, right? If so, it would be better if the note was phrased in those terms. And also used `class="note"`.